Mathlib/CategoryTheory/ConnectedComponents.lean

```diff
@@ -143,4 +168,6 @@ instance : (decomposedTo J).IsEquivalence where
 def decomposedEquiv : Decomposed J ≌ J :=
   (decomposedTo J).asEquivalence
 
+
+
```

Comment:
There are two extra blank lines here that you need to remove (I think that was my suggestion which is marked as resolved here)